frontend/src/components/UploadPreviewTable/UploadPreviewTable.tsx

```diff
@@ -38,17 +40,27 @@ export function UploadPreviewTable({
 
       {
         id: "actions",
-        accessor: () => (
-          // TODO add link
+        accessor: (p: PreviewRow) => (
           <td className="ednaTable__cell uploadPreviewTable__actions">
-            <Button type="link" disabled={!viewEnabled}>
-              View
-            </Button>
+            <Link
+              to={{
+                pathname: "/dashboard",
+                state: { projectId, compoundId: p.compoundId, targetId: p.targetId },
+              }}
+            >
+              <Button
+                propagate
+                type="link"
+                disabled={!projectId || !isDefined(p.compoundId) || !isDefined(p.targetId)}
```

Comment:
Is it possible

Reply:
it's, if compound or target are new

Reply:
also, I pass
`projectId={excelFile.state === "added" ? currentProject?.id : undefined}`
in `UploadPage.tsx`, so until a user added an experiment I pretend it doesn't have project (kostil)